pkg/model/network.go

```diff
@@ -276,6 +277,7 @@ func (b *NetworkModelBuilder) Build(c *fi.ModelBuilderContext) error {
 				Subnet:               utilitySubnet,
 				ElasticIP:            eip,
 				AssociatedRouteTable: b.LinkToPrivateRouteTableInZone(zone),
+				Tags:                 tags,
```

Comment:
Before I forget, I know @robinpercy will review, but @justinsb had to setup where we had to not just set the tags, but had to jump through a couple of hoops. See https://github.com/kubernetes/kops/pull/4572

I do not know if this PR actually needs that or not ...